### PR TITLE
feat: Add `logIframeErrors` option to not print iframe error to console

### PR DESCRIPTION
**Pull request by straker**
This uses the same option name as `axe-cli` with the idea being that it can just pass the option directly into this. The default is true so behavior doesn't change from before, and only passing `stdout: false` will cause it to throw the error.

Closes: #41 

## Reviewer checks

**Required fields, to be filled out by PR reviewer(s)**
- [x] Follows the commit message policy, appropriate for next version
- [x] Has documentation updated, a DU ticket, or requires no documentation change
- [x] Includes new tests, or was unnecessary
- [x] Code is reviewed for security by: @WilcoFiers

### CODE REVIEW

File: test/unit/axe-injector.js [hunk 1]
```diff
@@ -215,5 +224,23 @@ describe('AxeInjector', () => {
...
+      const stub = sinon.stub(injector, 'injectIntoAllFrames');
+
+      stub.rejects(new Error('oh no!'));
+
+      injector.inject(err => {
+        if (err) {
+          return done();
+        }
+
+        done('injector did not pass error to callback');
```
Comment (stephenmathieson) [inline test/unit/axe-injector.js]: Mocha's `done` callback does not accept a string. Please pass an error here.

**Comment by straker** [comment]
@stephenmathieson the option mimics the [axe-cli `--stdout` flag](https://github.com/dequelabs/axe-cli#sending-results-to-stdout), which is a boolean. I was going for consistency in API since axe-cli uses axe-webdriver.

**Comment by stephenmathieson** [comment]
Maybe something like `logIframeErrors: true | false`? This way it clearly states its purpose.

File: lib/axe-injector.js [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
-  constructor({ driver, config, axeSource = null }) {
+  constructor({ driver, config, axeSource = null, options = {} }) {
     this.driver = driver;
     this.axeSource = axeSource || require('axe-core').source;
     this.config = config ? JSON.stringify(config) : '';
+    this.options = options;
+
+    // default logIframeErrors to true so it retains the original behavior
+    this.options.logIframeErrors =
+      typeof this.options.logIframeErrors !== 'undefined'
```
Comment (stephenmathieson) [inline lib/axe-injector.js]: Not a big deal, but why do we support non-boolean values here?

```js
new AxeInjector(driver, config, { logIframeErrors: 'sharks and potatoes' })
```

Comment (straker) [reply lib/axe-injector.js]: There's no type checking, so we support anything thats truthy and falsey?

**Comment by padmavemulapati** [comment]
Dev Task.

**Comment by stephenmathieson** [comment]
@padmavemulapati there is something "QA-able" here. One could verify the behavior with/without the `logIframeErrors` option provided.

**Pull request merged**